app/components/peopletable/peopletable.js

```diff
@@ -406,9 +389,9 @@ const PeopleTable = translate()(class PeopleTable extends React.Component {
 });
 
 PeopleTable.propTypes = {
-  people: React.PropTypes.array,
-  trackMetric: React.PropTypes.func.isRequired,
-  onRemovePatient: React.PropTypes.func.isRequired,
+  people: PropTypes.array,
+  trackMetric: PropTypes.func.isRequired,
+  onRemovePatient: PropTypes.func.isRequired,
 };
 
 module.exports = PeopleTable;
```

Comment:
`export default PeopleTable;`

And in the index.js file, change `module.exports = require('./peopletable');` to
```js
import PeopleTable from './peopletable';
export default PeopleTable;
```